ros/src/styx/bridge.py

```diff
@@ -10,7 +10,7 @@
 from sensor_msgs.msg import Image
 import sensor_msgs.point_cloud2 as pcl2
 from std_msgs.msg import Header
-from cv_bridge import CvBridge, CvBridgeError
+from cv_bridge import CvBridge  # , CvBridgeError
```

Comment:
Was commenting out `CvBridgeError` intentional?

Reply:
Yea, it's not used so I was just cleaning it up.  I'll just delete it instead.